catalog/spatial/geocoding/spatial-geocoding-admin-module/src/main/webapp/js/view/Ingest.view.js

```diff
@@ -0,0 +1,241 @@
+                        view.render();
+                  }
+                });
+                this.startPollingModel(view.model);
+                return false;
+            },
+            post_local_file: function(resourceUri, id, model) {
+                var checked = $("#create_index_local").is(':checked');
+                var view = this;
+
+                $.ajax({
+                  url: "/jolokia/exec/org.codice.ddf.spatial.admin.module.service.Geocoding:service=geocoding/execWithFilePath/" + id + "/" + resourceUri + "/" + checked,
+                  dataType: 'json',
+                  success: function(data) {
+                        if(data.value) {
+                            model.set('state', 'done');
+                        } else {
+                            model.set('state', 'failed');
+                        }
+                        view.setStartButton();
+                  },
+                  error: function() {
+                        model.set("state", "failed");
+                  }
+                });
+                this.startPollingModel(model);
+                return false;
+            },
+            startPollingModel(model) {
+                var view = this;
+
+                (function poll() {
+                    var currentProgress = model.get("progress") ;
+
+                    if(currentProgress !== 100 && model.get("state") !== "failed" && model.get("state") !== "failed-geonames" ) {
+                        setTimeout(function() {
+                            $.ajax({
+                                url : "/jolokia/exec/org.codice.ddf.spatial.admin.module.service.Geocoding:service=geocoding/progressCallback/",
+                                success: function(data) {
+                                    var progress = parseInt(data.value)
+                                    model.set('progress', progress);
+                                },
+                                error: function(data) {
+                                    model.set('progress', 100);
+                                    if(model.get('state') === 'uploading-geonames') {
+                                        model.set('state', 'failed-geonames');
+                                    } else {
+                                        model.set('state', 'failed');
+                                    }
+                                },
+                                dataType: "json",
+                                complete: poll
+                            });
+                        }, 500);
+                        view.render();
+                    }
+                })();
+            },
+            disableStartUploadButton: function(disable) {
+                this.$('.start-upload-local').prop('disabled', disable);
+            },
+            startUpload: function() {
+                this.collection.trigger('startUpload');
+                wreqr.vent.trigger('upload:start');
```

Comment:
So why are we pulling in wreqr and throwing an event that no one is listening to?